packages/xrpl/package.json

```diff
@@ -44,6 +44,7 @@
     "lodash": "^4.17.4",
     "node-polyfill-webpack-plugin": "^2.0.1",
     "react": "^18.2.0",
+    "run-s": "^0.0.0",
```

Comment:
I found no new usages of run-s.  I think this can be removed along with package-lock.json changes.

Reply:
run-s is used for `npm run build` - does it need to be a dev dependency for that?

Reply:
Good call.  Wonder why it wasn't there before.

Reply:
TODO: Re-add this since I removed it in another commit

Reply:
Re-added.